From 2ad3d2dc95239cc7d606a78559508133668cf20d Mon Sep 17 00:00:00 2001
From: Norvin Shuster <norvin.shuster@intel.com>
Date: Fri, 21 Jun 2013 15:30:38 -0700
Subject: [PATCH] [MERGE TO MAIN] Chaabi middleware Resolve build issues

BZ: 117874

Resolved the build issues with the android logging library changes
(use liblog instead of -llog).

Resolved a compiler warning in the miscutils library.

Removed android_runtime library from the SEP DRM JNI library it is not
needed.

Change-Id: Ie5a280f65339e24beef6b63b36e3f30e1a237db5
Signed-off-by: Norvin Shuster <norvin.shuster@intel.com>
---
 App/Android.mk               |    5 +++--
 Lib/Android.mk               |    4 ++--
 Lib/misc_utils.c             |    1 -
 service/sepdrmjni/Android.mk |    1 -
 4 files changed, 5 insertions(+), 6 deletions(-)

diff --git a/App/Android.mk b/App/Android.mk
index 1075bb8..80702d7 100644
--- a/App/Android.mk
+++ b/App/Android.mk
@@ -96,7 +96,7 @@ LOCAL_SRC_FILES += \
 libsecurity_api_stub.c
 
 LOCAL_STATIC_LIBRARIES := CC6_UMIP_ACCESS CC6_ALL_BASIC_LIB
-LOCAL_SHARED_LIBRARIES := libsecurity_api libc libcutils libcrypto
+LOCAL_SHARED_LIBRARIES := libsecurity_api libc libcutils libcrypto liblog
 
 
 LOCAL_C_INCLUDES := \
@@ -152,7 +152,7 @@ LOCAL_LIB_DIR := $(LOCAL_PATH)/../Lib
 LOCAL_SRC_FILES += \
 libsecurity_api_stub.c
 
-LOCAL_STATIC_LIBRARIES := libsecurity_api CC6_UMIP_ACCESS CC6_ALL_BASIC_LIB libc libcutils libcrypto_static
+LOCAL_STATIC_LIBRARIES := libsecurity_api CC6_UMIP_ACCESS CC6_ALL_BASIC_LIB libc libcutils libcrypto_static liblog
 
 
 LOCAL_C_INCLUDES := \
@@ -214,6 +214,7 @@ LOCAL_STATIC_LIBRARIES := libmiscutils		\
 	CC6_ALL_BASIC_LIB			\
 	libcutils				\
 	libc					\
+	liblog					\
 	libcrypto_static
 
 LOCAL_C_INCLUDES :=					\
diff --git a/Lib/Android.mk b/Lib/Android.mk
index a2bf69e..fac1539 100644
--- a/Lib/Android.mk
+++ b/Lib/Android.mk
@@ -219,7 +219,7 @@ LOCAL_SRC_FILES+= \
 security_api.c
 
 LOCAL_STATIC_LIBRARIES := CC6_UMIP_ACCESS CC6_ALL_BASIC_LIB
-LOCAL_SHARED_LIBRARIES := libcutils libc libcrypto
+LOCAL_SHARED_LIBRARIES := libcutils libc libcrypto liblog
 
 LOCAL_LDFLAGS += -llog
 
@@ -330,7 +330,7 @@ DX_APP_DIR:= $(LOCAL_PATH)
 LOCAL_SRC_FILES+= \
 security_api.c
 
-LOCAL_STATIC_LIBRARIES := CC6_UMIP_ACCESS CC6_ALL_BASIC_LIB libcutils libc libcrypto_static
+LOCAL_STATIC_LIBRARIES := CC6_UMIP_ACCESS CC6_ALL_BASIC_LIB liblog libcutils libc libcrypto_static
 
 LOCAL_LDFLAGS += -llog
 
diff --git a/Lib/misc_utils.c b/Lib/misc_utils.c
index 157fb57..5a7f9cd 100644
--- a/Lib/misc_utils.c
+++ b/Lib/misc_utils.c
@@ -55,7 +55,6 @@ int read_data_from_file( const char * const pFilename,
 {
 	int result = EXIT_FAILURE;
 	int status = 0;
-	off_t fileSizeInBytes;
 	struct stat fileInfo;
 	int inputFileHandle = 0;
 	ssize_t bytesRead = 0;
diff --git a/service/sepdrmjni/Android.mk b/service/sepdrmjni/Android.mk
index b878d5b..2b5d38b 100644
--- a/service/sepdrmjni/Android.mk
+++ b/service/sepdrmjni/Android.mk
@@ -35,7 +35,6 @@ LOCAL_SHARED_LIBRARIES := \
     libcutils \
     libutils \
     libbinder \
-    libandroid_runtime \
     libnativehelper \
     libsepdrm
 
-- 
1.7.4.1

